packages/stream_video_flutter/lib/src/call_screen/call_container.dart

```diff
@@ -82,7 +82,7 @@ class StreamCallContainer extends StatefulWidget {
   final VoidCallback? onCancelCallTap;
 
   /// The action to perform when the call is disconnected. By default, it pops the current route.
-  final VoidCallback? onCallDisconnected;
+  final void Function(DisconnectReason)? onCallDisconnected;
```

Comment:
Instead of just returning a `DisconnectReason` I think it's good to return a new class, something like `DisconnectProperties` or something better. We can start with only adding the reason, but we would be able to add more properties without another breaking change.

Reply:
Done